server/pages/index.js

```diff
@@ -319,6 +319,13 @@ module.exports = function() {
 
 	app.get( '/theme', ( req, res ) => res.redirect( '/design' ) );
 
+	app.get( '/plans', function( req, res, next ) {
```

Comment:
If we moved it to line 317 https://github.com/Automattic/wp-calypso/pull/7100/files#diff-b2d6784517d1151d7e75f0287d0c6478R317 , we could get rid of the development check on every page call.